frontend/ui/widget/menu.lua

```diff
@@ -1107,6 +1108,26 @@ function Menu:updateItems(select_number)
     end)
 end
 
+-- merge TitleBar layout into self FocusManager layout
+function Menu:addTitleBarLayout()
```

Comment:
This function name feels a bit odd (it reads as "add a title bar layout").
Shouldn't it read as `Menu:addTitleBarToLayout()` or `Menu:mergeTitleBarIntoLayout()`

Reply:
method renamed